package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
Don't need to change this with this PR. Should change the base to develop so we can PR to a release branch from there. We'll update the version on that PR